### PR TITLE
Readme documentation fe

### PR DESCRIPTION
**Pull request by aplasket**
PR Template:

# high level what you did:
-added readme and updated with FE info

## Issues?
- we still have a few things to add -- mainly wireframe photos, production website and logo photos! and to decide on if we're CivicVoicePlus or CivicVoice+

## Questions or Notes:
any questions you have for the reviewer

## Overall Checklist:
- [ ] tests written for all methods added
- [ ] test coverage above 90%
- [ ] RSpec running clean
- [ ] ran rails s -- features working properly!
- [x] pulled/merged milestone branch before submitting PR

### CODE REVIEW

File: CVP FE schema.png
Comment (Jredish11) [inline CVP FE schema.png:1]: hi lonely table

File: README.md [hunk 1]
```diff
@@ -1,24 +1,270 @@
...
+    <img src="images/logo.png" alt="Logo" width="80" height="80">
+  </a>
 
-* System dependencies
+<h3 align="center">CivicVoice+</h3>
 
-* Configuration
+  <p align="center">
+    Welcome to the front end repository for CivicVoicePlus!
+    CivicVoicePlus is a web application built for the Turing School of Software and Design's Mod 3 Consultancy project. Read more about project requirements: https://backend.turing.edu/module3/projects/consultancy/
```
Comment (Jredish11) [inline README.md:22]: hmm I think CivicVoice+ looks better but people could be mistaken and think its CivicVoice positive

File: README.md [hunk 2]
```diff
@@ -1,24 +1,270 @@
...
+<br>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- Technical Solutions -->
+## Technical Solutions:
+As part of the Consultancy project requirements, the CivicVoicePlus team challenged ourselves to stretch technologies during the 11-day design and development process. We selected the following based on the challenges we anticipated facing while building out our MVP, and adjusted our choices to reflect our individual and team learning goals as well as blockers that came up during the course of working on the project.
+
+### Caching
+* Challenge: Two of the API endpoints that expose data for the front end of our application rely on both database queries and external API calls, which significantly slowed down the processing time and resulted in a less-than-ideal experience for our users. Additionally, we were hitting rate limits on our API endpoints.
```
Comment (JCBeldo) [inline README.md:187]: Love this challenge/solution part :)

**Review by JCBeldo — APPROVED** [review]
Excellent job Ashley!  Thanks for doing this!

**Review by jcjurado3 — APPROVED** [review]
Thanks for doing this! This looks great!

**Pull request merged**